docs/mosviz/import_data.rst

```diff
@@ -12,8 +12,23 @@ Automatic Directory Loading
 Mosviz provides instrument-specific directory parsers for select instruments. At this
 time, Mosviz supports automatic parsing for the following instruments:
 
-* JWST NIRSpec
+* JWST NIRSpec (levels 2 and 3)
 * JWST NIRISS
+* JWST NIRCam
```

Comment:
Why only NIRSpec has `(levels 2 and 3)` specified? What about NIRISS and NIRCam?

Reply:
NIRISS and NIRCam only have level 2 available from what I've been told, there is no level 3.